src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -1181,15 +1049,21 @@ private SynthesizedSealedPropertyAccessor MakeSynthesizedSealedAccessor()
 
         #region Attributes
 
-        protected override IAttributeTargetSymbol AttributesOwner => this;
+        protected virtual IAttributeTargetSymbol AttributesOwner => this;
```

Comment:
>virtual [](start = 18, length = 7)

These three properties do not appear to be overridden and do not need to be `virtual`.